pkgs/top-level/all-packages.nix

```diff
@@ -18665,6 +18665,15 @@ in
 
   nginx = nginxStable;
 
+  nginxQuic = callPackage ../servers/http/nginx/quic.nix {
+    withPerl = false;
```

Comment:
I am curious why perl is disabled?

Reply:
Copied it from below (nginxStable), no idea